tests/test_read_write/test_read_csv.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from itertools import product
+from clumper import Clumper
+
+
+paths = ["tests/monopoly.csv", "https://calmcode.io/datasets/monopoly.csv"]
+nrows = [(None, 22), (10, 10), (15, 15)]
```

Comment:
I might add one more instance here. 

If the dataset has 22 rows but we ask for 50, then we should still get only 22.